src/DropNCrop.js

```diff
@@ -120,56 +126,56 @@ class DropNCrop extends Component {
 
     return (
       <div className={classNames(dropNCropClasses)}>
-        {value && value.src
-          ? <Cropper
-              ref={input => {
-                this.cropperRef = input;
-              }}
-              src={value && value.src}
-              style={{
-                height: canvasHeight,
-                width: canvasWidth,
-              }}
-              cropend={this.onCrop} // Only use the cropend method- it will reduce the callback/setState lag while cropping
-              {...cropperOptions}
-            />
-          : <Dropzone
-              className="dropzone"
-              activeClassName="dropzone--active"
-              onDrop={this.onDrop}
-              style={{
-                height: canvasHeight,
-                width: canvasWidth,
-              }}
-            >
-              <div key="dropzone-instructions">
-                {!instructions
-                  ? <div className="dropzone-instructions">
-                      <div className="dropzone-instructions--main">
-                        {customMessage.instructions}
-                      </div>
-                      <div className="dropzone-instructions--sub">
-                        {customMessage.acceptedFileTypes}
-                        {' '}
-                        {allowedFileTypes
-                          .map(mimeType => `.${mimeType.split('/')[1]}`)
-                          .join(', ')}
-                      </div>
-                      <div className="dropzone-instructions--sub">
-                        {customMessage.maxFileSize} {bytesToSize(maxFileSize)}
-                      </div>
-                    </div>
-                  : instructions}
-              </div>
-              {value && value.error
-                ? <div
-                    key="dropzone-validation"
-                    className="dropzone-validation"
-                  >
-                    {value && value.error}
+        {value && value.src ? (
```

Comment:
Can you explain these changes? Did you run prettier on this or something?

When I'm reviewing a PR having large blocks of formatting changes to unrelated code makes it harder for me to understand 100% of what's going on.

Reply:
prettier ran automatically, so i'd assumed it was intended that way. let me recommit but without prettier